sentry-core/src/main/java/io/sentry/core/Sentry.java

```diff
@@ -49,7 +51,16 @@ public static boolean isEnabled() {
 
   /** Initializes the SDK */
   public static void init() {
-    init(new SentryOptions());
+    init(GLOBAL_HUB_DEFAULT_VALUE);
+  }
+
+  /**
+   * Initializes the SDK
+   *
+   * @param globalHubMode the globalHubMode
+   */
+  public static void init(boolean globalHubMode) {
```

Comment:
@marandaneto can we remove this overload? I'd rather we have this flag less exposed. It'd be fine to only be able to flip it on the overload with all parameters.